database/postgres/seed.js

```diff
@@ -93,4 +93,4 @@ const seed = async (numOfRecords = 1000, batchSize = 1000) => {
   }
 }
 
-seed(100000, 1000);
\ No newline at end of file
+seed(1000000, 1000);
```

Comment:
is 1000 the maximum allowed batch size or is it just a nice round number that you picked?

Reply:
I made a batch size variable just to experiment. After a number of tests inserting 100K and 1M records at a time, 700-1000 range seemed to be the fastest for some reason.